cmd/kubelet/app/options/options.go

```diff
@@ -96,6 +96,9 @@ func (s *KubeletServer) AddFlags(fs *pflag.FlagSet) {
 	fs.Var(componentconfig.IPVar{Val: &s.Address}, "address", "The IP address for the Kubelet to serve on (set to 0.0.0.0 for all interfaces)")
 	fs.Int32Var(&s.Port, "port", s.Port, "The port for the Kubelet to serve on.")
 	fs.Int32Var(&s.ReadOnlyPort, "read-only-port", s.ReadOnlyPort, "The read-only port for the Kubelet to serve on with no authentication/authorization (set to 0 to disable)")
+	fs.BoolVar(&s.TLSCertAuth, "tls-cert-auth", s.TLSCertAuth, "Enable TLS client authentication")
+	fs.StringVar(&s.TLSCAFile, "tls-ca-file", s.TLSCAFile, ""+
```

Comment:
The client cert CA should be separately specified. The CA bundle for the master might not be the CA desired for client certs (a narrower intermediate might be desired). Matching the arg used for the client ca on the master would make sense to me